spyder/plugins/editor/tests/test_plugin.py

```diff
@@ -143,12 +143,8 @@ def test_renamed_tree(editor_plugin, mocker):
 
     editor.renamed_tree('/test/directory', '/test/dir')
     assert editor.renamed.call_count == 3
-    assert editor.renamed.called_with(source='/test/directory/file1.py',
-                                      dest='test/dir/file1.py')
-    assert editor.renamed.called_with(source='/test/directory/file2.txt',
-                                      dest='test/dir/file2.txt')
-    assert editor.renamed.called_with(source='/test/directory/file4.rst',
-                                      dest='test/dir/file4.rst')
+    editor.renamed.assert_called_with(source='/test/directory/file4.rst',
+                                      dest='/test/dir/file4.rst')
```

Comment:
This is clearly not a fix for this test because we need to check that all files have been renamed. And that's detected by our Windows tests.

So, I recommend instead to revert these changes and simply skip it in your test env.

Reply:
Actually, this test didn't do anything (even on Windows), because `called_once` always silently returned a true value without testing anything.  I've just pushed a new version of the patch, replacing the `assert_called_with()` call, which only tests the most recent call, with `assert_any_call()`, which checks whether the method has ever been called with the specified arguments.  So this new version of the patch now correctly checks for all three calls.

Reply:
Small clarification to that comment: `called_once()` always silently returned a true value on Python <= 3.11.  But on Python 3.12, it throws an error, which is how I discovered this bug.